src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -44,9 +44,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _lastMouseClickPos{},
         _selectionNeedsToBeCopied{ false }
     {
+        _guid = ::Microsoft::Console::Utils::CreateGuid();

```

Comment:
Just an idea, but instead of using GUIDs you can also use an atomic counter. That might make debugging easier because the numbers will be much shorter initially and make identifying them easier since they're strictly sequential.

Reply:
that's not a bad idea. That'll take me a bit, the `guid` kinda propogates a lot through the subsequent PRs. Lemme make sure that all works